perl-package/test.sh

```diff
@@ -0,0 +1,16 @@
+MXNET_HOME=$1
```

Comment:
For MXNET_HOME and HOME, why do you choose /workspace and /workspace/ut-perl-gpu, respectively?

As far as I know /workspace should only contain workspaces for each stage in this Jenkinsfile, each containing a copy of mxnet source code. But for example, you're referencing ${MXNET_HOME}/lib -> /workspace/lib here which I don't think exists.